Main.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class Main {
+    static Scanner userInputScanner = new Scanner(System.in);
+    static ControlPanel UserInputSelectOptionPanel = new ControlPanel();
+
+    public static void main(String[] args) {
```

Comment:
it could be more level menu like:
1. Circle
     - Area calculation
     - Arc calculation
2. Rectangle
    - Area calculation
    - Diagonal calculation
3. exit